docs/source/fetching/persisted-queries.mdx

```diff
@@ -0,0 +1,160 @@
+- a `path` to output the manifest file
+- the `version`: either `persistedQueries` for persisted queries or `legacyAPQs` for APQs
+
+```json title="apollo-codegen-config.json"
+"output": {
+    "operationManifest" : {
+        "path" : "./generated/operationIdentifiers.json",
+        "version" : "persistedQueries"
+    }
+}
+```
+
+2. In the `options` section of your configuration, set the `operationDocumentFormat` array to `definition`, `operationId`, or both `definition` and `operationId`. This determines how to generate the operation documents for your generated operations.
+
+```json title="apollo-codegen-config.json"
+"options": {
+    "operationDocumentFormat" : [
+        "definition",
+        "operationId"
+    ]
+}
+```
+
+Once you've configured code generation, you generate an operation manifest in one of two ways:
+- using the standard [`generate`](./../code-generation/codegen-cli#generate) command 
+- using the [`generate-operation-manifest`](./../code-generation/codegen-cli#generate-operation-manifest) command; this generates the operation manifest without running full code generation
+
+The resulting operation manifest for `persistedQueries` looks something like this: 
+
+```json title="operationIdentifiers.json"
+{
+  "format": "apollo-persisted-query-manifest",
+  "version": 1,
+  "operations": [
+    {
+      "id": "e0321f6b438bb42c022f633d38c19549dea9a2d55c908f64c5c6cb8403442fef",
+      "body": "query GetItem { thing { __typename } }",
+      "name": "GetItem",
+      "type": "query"
+    }
+  ]
+}
+```
+
+The resulting operation manifest JSON for `legacyAPQ` looks something like this: 
+
+```json title="operationIdentifiers.json"
+{
+  "525cd90917f8945d8d6a68fde991aaf766ff7d7b227ab2cd8dc97b7959c8d0d6" : {
+    "name": "GetItem",
+    "source": "query GetItem { thing { __typename ...Name } } fragment Name on Item { __typename name }"
+  }
+}
+```
+
+To automatically update the manifest for each new app release, you can include the [`generate`](./../code-generation/codegen-cli#generate) or [`generate-operation-manifest`](./../code-generation/codegen-cli#generate-operation-manifest) command in your CI/CD pipeline.
+
+#### Publish manifests to the PQL
+
+<blockquote>
+
+**This step is only required for persisted queries, not APQ.**
+
+Ensure your Rover CLI version is `0.17.2` or later. Previous versions of Rover don't support publishing operations to a PQL.
+[Download the latest version.](/rover/getting-started/)
+
+</blockquote>
+
+If you want to preregister your manifest as a safelist of trusted operations , you need to publish it to a PQL using the [Rover CLI](/rover/):
+
+```bash title="Example command"
+rover persisted-queries publish my-graph@my-variant \
+  --manifest ./persisted-query-manifest.json
+```
+
+- Replace `my-graph@my-variant` with the **graph ref** of _any_ variant your PQL is [linked to](#12-link-the-pql-to-variants). Your platform team can provide this for you.
```

Comment:
`[linked to](#12-link-the-pql-to-variants)`

Looks like this is supposed to link to an anchor in another page that this line was copied from. But this isn't going to be a valid link.

I think this stuff is all the more reason to put this section in a shared block referenced by all the platforms if possible.

Reply:
Very true, thanks for catching.